### PR TITLE
 [devtools-translate] Portuguese - What's New in DevTools (Chrome 96) #1618

### LINKED ISSUE #1618: [devtools-translate] Portuguese - What's New in DevTools (Chrome 96)
Post: https://developer.chrome.com/blog/new-in-devtools-96/

Translator: @alvarocamillont
Reviewer: @khaosdoctor

Stable rollout: Nov 16
Translation ready by: Nov 7

### PR DESCRIPTION
**Pull request by alvarocamillont**
Fixes #1618

Changes proposed in this pull request:

    Add portuguese translations for What's New in DevTools M96

### CODE REVIEW

File: site/pt/blog/new-in-devtools-96/index.md [hunk 1]
```diff
@@ -29,166 +29,195 @@ draft: true
...
-
+## Prévia de funcionalidade: novo painel de visão geral CSS  {: #css-overview }
 <!-- Use the new **CSS Overview** panel to identify potential CSS improvements on your page.
 [Open the **CSS Overview** panel](/docs/devtools/css-overview#open), then click on **Capture overview** to generate a report of your page’s CSS. -->
 
+Use o novo painel **Visão geral CSS** para identificar possíveis melhorias de CSS em sua página.
+[Abra o painel **Visão geral CSS**](/docs/devtools/css-overview#open) e clique em **Capture visão geral** para gerar um relatório do CSS da sua página.
+
 <!-- You can further drill down on the information. For example, click on a color in the **Colors** section to view the list of elements that apply the same color. Click on an element to open the element in the **Elements** panel. -->
+Você pode detalhar ainda mais as informações. Por exemplo, clique em uma cor na seção **Cores** para visualizar a lista de elementos que aplicam a mesma cor. Clique em um elemento para abri-lo no painel **Elementos**.
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-96/index.md:40]: Wondering if "expandir" reads better as a translation for drill down than "detalhar".
```suggestion
Você pode expandir as informações. Por exemplo, clique em uma cor na seção **Cores** para visualizar a lista de elementos que aplicam a mesma cor. Clique em um elemento para abri-lo no painel **Elementos**.
```

File: site/pt/blog/new-in-devtools-96/index.md [hunk 2]
```diff
@@ -29,166 +29,195 @@ draft: true
...
 Issue relacionada: [1254557](https://crbug.com/1254557)
 
 
 <!-- ## Rendering tab updates  -->
-
+## Atualizações da guia de renderização
 <!-- ### Emulate the CSS prefers-contrast media feature {: #prefers-contrast } -->
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/47fsHvVLiVC9J0eWY9wD.png", alt="Emulate the CSS prefers-contrast media feature", width="800", height="483" %}
+### Emule o recurso de contraste preferencial para mídia de CSS {: #prefers-contrast }
```
Comment (andreban) [inline site/pt/blog/new-in-devtools-96/index.md:59]: `prefers-contrast` is the name of the CSS media (https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast), used in the stylesheet. I'm wondering we should not translate the term.

**Review by andreban — CHANGES_REQUESTED** [review]
Overall, looks good, thank. Requested a couple changes.

File: site/pt/blog/new-in-devtools-96/index.md [hunk 3]
```diff
@@ -29,166 +29,213 @@ draft: true
...
+Leia [este artigo](/docs/devtools/css-overview) para saber mais no painel **Visão geral CSS**.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="CSS Overview panel", width="800", height="509" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="Painel Visão Geral CSS", width="800", height="509" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/ef26abc89035075bbdb08f1b26c1b8fd942ffc04 #}
 
 Issue relacionada: [1254557](https://crbug.com/1254557)
 
+<!-- ## Restored and improved CSS length edit and copy experince {: #length } -->
```
Comment (chybie) [inline site/pt/blog/new-in-devtools-96/index.md:54]: Hi @alvarocamillont , I added a new section here (a fix we recently merged back to M96). Sorry for late inform!

**Review by chybie — CHANGES_REQUESTED** [review]

**Comment by chybie** [comment]
@googlebot I consent.

**Review by chybie — APPROVED** [review]

**Review by andreban — APPROVED** [review]
LGTM, Thank you!

**Pull request merged**